libraries/standard/mqtt/src/iot_mqtt_serialize.c

```diff
@@ -2059,7 +2059,11 @@ IotMqttError_t _IotMqtt_SerializePingreq( uint8_t ** pPingreqPacket,
                                           size_t * pPacketSize )
 {
     /* PINGREQ packets are always the same. */
-    static const uint8_t pPingreq[ MQTT_PACKET_PINGREQ_SIZE ] =
+    /* It is not necessary to make this array const. Since there are other
+     * types of MQTT packets that are not constant, this array would be
+     * cast to remove the const qualifier anyway. */
+    /* coverity[misra_c_2012_rule_8_13_violation] */
+    static uint8_t pPingreq[ MQTT_PACKET_PINGREQ_SIZE ] =
```

Comment:
The 11.8 violation occurred on line 2073 ( removing `const`-ness of `pPingreq` by casting to `uint8_t *` ). After some offline discussion, I decided the best way to deal with this would be to remove the `const` from the array declaration, since adding `const` anywhere to `pPingreqPacket` would require changing the function signature, which the serializer overrides rely on.